hieradata/node/auxtel-hcu02.cp.lsst.org.yaml

```diff
@@ -0,0 +1,67 @@
+---
+nm::connections:
```

Comment:
This is OK as-is but please switch to using the yaml style config instead of heredoc'd strings in the future.  I have a script to convert this style to the pure yaml style.

Reply:
Copy that.